### ISSUE TITLE
Prototype HUD screen

### ISSUE THREAD

**Issue by fnoop** [goodrobots/maverick-web#42]
Good example of HUD screen:
https://discuss.ardupilot.org/t/new-mp-hud-i-have-been-working-on/26602

**Comment by fnoop** [comment]
Artificial horizon is the obvious starting point, but should it be a fighter-like HUD, or a passenger-plane-like separate instrument?  Or option of both?

**Comment by SamuelDudley** [comment]
Personally I like the fighter style HUD, but the importance should be readability and functionality over prettiness

**Comment by fnoop** [comment]
These are awesome skeuomorphic dials
http://sebmatton.github.io/flightindicators/

**Comment by fnoop** [comment]
https://github.com/eschnou/ardrone-webflight

**Comment by SamuelDudley** [comment]
That ardrone-webflight software is interesting hits an lot of points we were discussing.

**Comment by SamuelDudley** [comment]
https://youtu.be/JzRRk1U7fiA
What's nice is that the AP setpoint and current values can be clearly seen. RC stick inputs are a nice touch too.

**Comment by fnoop** [comment]
https://github.com/richtr/Artificial-Horizon/blob/master/script/main.js

**Comment by cglusky** [comment]
I would not worry about skeuomorphic dials.  I think it's safe to assume that is a learned interface for pilots and most sUAS operators will not have that background.

**Comment by fnoop** [comment]
Yeah i've ditched the skeuomorphic stuff!  Just a HUD overlay for now

**Comment by fnoop** [comment]
I've opted to put a lot of the info in dials into simple text indicators in the top bar.  This is a lot neater on the much smaller interfaces we use with drones (eg. tablet instead of a real cockpit..), and much easier to read for non-trained pilots (eg. me).
